mmv1/products/cgc/terraform.yaml

```diff
@@ -16,6 +16,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   Snippet: !ruby/object:Overrides::Terraform::ResourceOverride
     cgc_only: true
     examples:
+    # Compute
+      - !ruby/object:Provider::Terraform::Examples
+        name: "flask-google-cloud-quickstart"
```

Comment:
@betsy-lichtenberg the filename and reference to it here needs to be snake case. We use this name as-is in our code, and the hyphens cause build errors